lib/deimos/utils/schema_class.rb

```diff
@@ -10,10 +10,23 @@ class << self
         # @return [Array<String>]
         def modules_for(namespace)
           modules = ['Schemas']
-          namespace_folder = namespace.split('.').last
-          if Deimos.config.schema.generate_namespace_folders && namespace_folder
-            modules.push(namespace_folder.underscore.classify)
+
+          if Deimos.config.schema.generate_namespace_folders
+            namespace_map = Deimos.config.schema.schema_namespace_map.sort_by{|k,v| -k.length}.to_h
+            namespace_override = namespace_map.keys.detect{ |k| namespace.include?(k) }
+
+            module_namespace = if namespace_override.present?
+              # don't use Schemas default module
+              modules = Array(namespace_map[namespace_override].dup)
```

Comment:
```suggestion
              modules = [namespace_map[namespace_override]]
```

Reply:
I want the array constructor so that you don't have to provide an array in the config when you just want a single module ('com.org.suborg' => 'SchemaClasses') and so that it is also compatible with multiple modules ('com.org.suborg' => ['SchemaClasses', 'SubOrg']).

The `dup` was necessary but I removed it in [c80da4b](https://github.com/flipp-oss/deimos/pull/208/commits/c80da4b4907ba49ef921888a8a497e8e12e46853) as this change meant it was no longer needed